EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/77107
+factinate.com#?#.advertisment_notice
```

Comment:
What about:
`factinate.com#$#.advertisment_notice { position: absolute !important; left: -3000px !important; }`
instead of `#?#` rule?
As far as I know, extended CSS rules (`#?#`) are slower (and need more resources), so if possible I prefer to use just CSS rules.

By the way, it seems that there is an ad leftover on home page.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/110779804-46236f80-8264-11eb-9658-896abbb96584.png)

</details>

For example:
`factinate.com#$#.home-page-ad { position: absolute !important; left: -3000px !important; }`

Reply:
As you wish.

Reply:
Btw, @AdamWr Where did you the position and left: -3000px? I don't see it.'


![image](https://user-images.githubusercontent.com/76908410/110782041-9967e300-82a1-11eb-8d37-7fc620009327.png)

Reply:
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/110782807-e038e700-8267-11eb-8b6d-fe4af1ca4c32.png)

</details>